index.js

```diff
@@ -313,7 +317,9 @@ export default class ModalSelector extends React.Component {
                 visible={this.state.modalVisible}
                 onRequestClose={this.close}
                 animationType={this.props.animationType}
-                onDismiss={() => this.state.changedItem && this.props.onChange(this.state.changedItem)}
+                onDismiss={() => {
```

Comment:
appreciate the formatting tidiness, but since there's no functional change, please revert this.